bombastic/api/src/server.rs

```diff
@@ -278,13 +279,13 @@ async fn publish_sbom(
     let typ = verify_type(content_type)?;
     let enc = verify_encoding(req.headers().get(CONTENT_ENCODING))?;
     let id = &params.id;
-    let mut payload = payload.map_err(|e| match e {
-        PayloadError::Io(e) => e,
-        _ => io::Error::new(io::ErrorKind::Other, e),
+    let payload = payload.map_err(|e| match e {
+        PayloadError::Io(e) => StorageError::Io(e),
+        _ => StorageError::Io(io::Error::new(io::ErrorKind::Other, e)),
     });
```

Comment:
Should this have a ? to return the Err?

Reply:
I don't think so. It's just mapping of `PayloadError` to `StorageError`

Reply:
I guess my question is `payload` could be either `Ok(...)` or `Err(...)` at the end of the line, yeah? And then it is `.put_stream(...)` into the storage.  Even if it's an `Result::Err`?

Reply:
Ahhh